demos/const_torque_control.cpp

```diff
@@ -36,10 +35,9 @@ void ConstTorqueControl::loop()
   while(!stop_loop_)
   {
     time_logger.tic();
-    local_time = count * 0.001;
 
     // compute the control
-    for(int i=0; i<motor_list_.size() ; ++i)
+    for(unsigned int i=0; i<motor_list_.size() ; ++i)
```

Comment:
generally, shouldn't we use size_t for that (instead of unsigned?)

Reply:
That's actually a good question... I am not sure.

Reply:
i guess it depends actually what we do inside the for loop: https://stackoverflow.com/questions/1951519/should-i-use-stdsize-t-or-int-in-my-for-loops

Reply:
I like the answer, so basically get the type of the comparison constant. size_t for any vector style